flytekit/core/data_persistence.py

```diff
@@ -293,6 +293,14 @@ class FileAccessProvider(object):
     This is the class that is available through the FlyteContext and can be used for persisting data to the remote
     durable store.
     """
+    INSTANCE = 0
```

Comment:
why?

Reply:
is this only for debugging? if so lets delete this

Reply:
will do